src/tutorials/quick-introduction.md

```diff
@@ -690,7 +690,7 @@ function collatz_histogram(xs, executor = ThreadedEx())
                 fill!(view(hist, l+1:m), 0)
             end
             # Merge `obs` into `hist`:
-            @floop for (k, v) in pairs(obs)
```

Comment:
It seems nested `@floop` has an issue that introduces `Box`es. Working around this for now just by using the native `for` loop.

Reply:
https://github.com/JuliaFolds/FLoops.jl/issues/52